docs/user-guide/persistent-volumes/index.md

```diff
@@ -43,7 +43,7 @@ When a user is done with their volume, they can delete the PVC objects from the
 
 ### Reclaiming
 
-The reclaim policy for a `PersistentVolume` tells the cluster what to do with the volume after it has been released.  Currently, volumes can either be Retained or Recycled.  Retention allows for manual reclamation of the resource.  For those volume plugins that support it, recycling performs a basic scrub (`rm -rf /thevolume/*`) on the volume and makes it available again for a new claim.
+The reclaim policy for a `PersistentVolume` tells the cluster what to do with the volume after it has been released.  Currently, volumes can either be Retained, Recycled or Deleted.  Retention allows for manual reclamation of the resource.  For those volume plugins that support it, deletion removes both the `PersistentVolume` object from Kubernetes as well as associated storage asset in cloud infrastructure and recycling performs a basic scrub (`rm -rf /thevolume/*`) on the volume and makes it available again for a new claim.
```

Comment:
> "... asset in cloud infrastructure. Recycling performs..."

The "For those volume plugins that support" part should clearly apply to both recycling and deletion. I split it into two sentences, both explicitly mentioning plugin support.